### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by AnaisUrlichs**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (bowenwang1996) [inline .github/ISSUE_TEMPLATE/bug_report.md]: How is smartphone relevant?

Comment (AnaisUrlichs) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good question -- e.g. website, docs and products may have bugs on smartphone that are not visible on the browser in terms of UI/UX

Comment (bowenwang1996) [reply .github/ISSUE_TEMPLATE/bug_report.md]: But that is not relevant to nearcore.

Comment (AnaisUrlichs) [reply .github/ISSUE_TEMPLATE/bug_report.md]: So I will create separate templates for the other repos

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (evgenykuzyakov) [inline .github/ISSUE_TEMPLATE/bug_report.md]: @bowenwang1996 please adjust
Version:
- nearcore commit/branch
- rust version (if local)
- docker (if using docker)
- production/testnet/local

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Thank you for taking the time to file an issue, we highly appreciate your contribution.
+
+Before you go ahead, please make sure that the issue has not been reported already.
+For support questions, please head over to our [Discord](http://near.chat/) or ask on [StackOverflow](https://stackoverflow.com/search?q=nearprotocol).
+-->
+
+**Describe the bug**
+Please provide a short description of the bug.
+
+**To Reproduce**
```
Comment (bowenwang1996) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It feels like this part is for something frontend related and doesn't really apply here

Comment (AnaisUrlichs) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Some people might feel more comfortable to describe the bug? Could you change the template to better fit your needs?

**Pull request merged**